DTS-STN/Scrum-Poker
# Updated copy button

## [DUMS-84 - Update copy room id component](https://jira-dev.bdm-dev.dts-stn.com/browse/DUMS-84) (Jira Issue)

### Description

This PR updates the copy button within the `RoomInfo` component to show a tooltip upon copying in order to let the user know that the id has been copied to their clipboard. The button text has also been replaced with a copy SVG as per Figma designs.

### What to test for/How to test

1. Pull in branch
2. Type `npm ci` and `npm run dev`
3. Create a room (or join an existing one)
4. Click the copy button; A tooltip should pop up above the button saying `Copied!` and then fade away after a delay
5. Paste the code to ensure it has been copied properly
